### PR TITLE
Fixed a bug in the CorDapp loader that crashed when plugins were loaded

### PR DESCRIPTION
**Pull request by Clintonio**
Note: V1 bug fix.

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt [hunk 1]
```diff
@@ -134,7 +134,11 @@ class CordappLoader private constructor(private val cordappJarPaths: List<URL>)
     }
 
     private fun findPlugins(cordappJarPath: URL): List<CordaPluginRegistry> {
-        return ServiceLoader.load(CordaPluginRegistry::class.java, URLClassLoader(arrayOf(cordappJarPath), null)).toList()
+        println("HI")
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt]: Remove debug print statements

Comment (Clintonio) [reply node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt]: Fixed!

File: node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt [hunk 2]
```diff
@@ -134,7 +134,9 @@ class CordappLoader private constructor(private val cordappJarPaths: List<URL>)
     }
 
     private fun findPlugins(cordappJarPath: URL): List<CordaPluginRegistry> {
-        return ServiceLoader.load(CordaPluginRegistry::class.java, URLClassLoader(arrayOf(cordappJarPath), null)).toList()
+        return ServiceLoader.load(CordaPluginRegistry::class.java, URLClassLoader(arrayOf(cordappJarPath), appClassLoader)).toList().filter {
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt]: You don't need `toList`

Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/internal/cordapp/CordappLoader.kt]: Perhaps just a small comment on what the filter is doing

**Pull request merged**